src/main/java/cn/nukkit/command/defaults/GiveCommand.java

```diff
@@ -17,22 +17,11 @@ public GiveCommand(String name) {
         super(name, "%nukkit.command.give.description", "%nukkit.command.give.usage");
         this.setPermission("nukkit.command.give");
         this.commandParameters.clear();
-        this.commandParameters.put("toPlayerById", new CommandParameter[]{
+        this.commandParameters.put("default", new CommandParameter[]{
```

Comment:
Could you please return deleted options, so an item can be given both by id and by name?